TaskHouseApi/Persistence/Repositories/Repository.cs

```diff
@@ -66,33 +66,45 @@ public void UpdatePart(T baseModel, string[] nameOfPropertysToIgnore)
             bool IsModified = false;
             foreach (PropertyInfo property in propertyInfos)
             {
-                /// If the as ICollection or name is Id do nothing
-                if (property.GetValue(baseModel) is ICollection || property.Name == "Id")
+
+                /// If the name is Id or the propertyname should be ignored or the type of a Reference do nothing 
+                if (property.Name == "Id" || nameOfPropertysToIgnore.Contains(property.Name) || property.PropertyType == typeof(Reference))
```

Comment:
Do not know the cs way of doing this, but in js i would newline the if statement to ease the readability.

Reply:
```cs
if (
	property.Name == "Id" || 
	nameOfPropertysToIgnore.Contains(property.Name) || 
	property.PropertyType == typeof(Reference)
)
```